src/force_fields_input.F

```diff
@@ -671,6 +689,54 @@ SUBROUTINE read_quip_section(nonbonded, section, start)
       END DO
    END SUBROUTINE read_quip_section
 
+! **************************************************************************************************
+!> \brief Reads the NEQUIP section
+!> \param nonbonded ...
+!> \param section ...
+!> \param start ...
+!> \author teo
```

Comment:
I don't think you have to credit Teo here. Feel free to put you own name throughout this PR. It also helps with posterity to know who is the actual expert for a piece of code.

Reply:
thanks, changed.